[WIP] Test visitor sees a navbar [#145557787]

samlandfried opened the pull request:
@glassjoseph @jk1dd @bschwartz10 Is Danny not a collaborator on this project? Did he cut and run? Anyways, are both these "Finishes" in brackets necessary? And this dastardly test is ready to be implemented.
[Finishes #145557787]

samlandfried committed:
Test visitor sees a navbar

[Finishes #145557787]

dannyradden commented:
I think I had just forgotten to accept the invite! I dont think you put Finishes until the final commit. You just put the ID# in brackets.

bschwartz10 committed:
add test for visitor clicks on home button in navbar

bschwartz10 committed:
Merge pull request #4 from glassjoseph/visitor-clicks-on-home-button

[WIP]add test for visitor clicks on home button in navbar

bschwartz10 committed:
Add navbar for visitors ...

Currently 2 tests fail because they rely on the root path
that will be implemented by someone else.
[#145558485]

bschwartz10 commented:
@dannyradden This should work when we merge in the root path changes
